rubix-spi/src/main/java/com/qubole/rubix/spi/fop/ObjectPool.java

```diff
@@ -78,8 +76,11 @@ public void registerHost(String host)
     }
     log.debug(this.name + " : Borrowing object for partition: " + host);
     for (int i = 0; i < 3; i++) { // try at most three times
```

Comment:
Remove this retry, better exit fast and fallback to direct reads.

Reply:
removed.